mix.exs

```diff
@@ -168,6 +166,7 @@ defmodule EMQXUmbrella.MixProject do
       compiler: :permanent,
       runtime_tools: :permanent,
       redbug: :permanent,
+      xmerl: :permanent,
```

Comment:
This changes the order of application loading/starting.

Perhaps keep it at the end like the rebar version?

I think the CI check missed this and the license/enterprise_conf problem.  Trying to fix it here: https://github.com/emqx/emqx/pull/8058

Reply:
I changed both sides, so the CI won't complain (not for this app).